src/dashboard/DashboardComponent.vue

```diff
@@ -51,32 +42,17 @@ div.container
 
 
       h3 Donation history
-      div(v-if="donationList.length === 0")
-        | No donations made so far
-      b-card.p-2.bt-0(v-else no-body)
-        table.table.table-sm(style="overflow: hidden; table-layout: fixed")
-          tr
-            th(style="width: 25%; border-top: 0") Date
-            th(style="border-top: 0") Creator
-            th.text-right(style="width: 20%; border-top: 0") Amount
-          tr(v-for="donation in donationList")
-            td(style="white-space: nowrap; text-overflow: ellipsis; overflow: hidden;")
-              // TODO: Adjust this to also work on mainnet when we have a DEBUG variable
-              a(:href="'https://ropsten.etherscan.io/tx/' + donation.transaction" target="_blank")
-                | {{donation.date.toLocaleDateString()}}
-            td(style="white-space: nowrap; text-overflow: ellipsis; overflow: hidden;")
-              a(:href="donation.url" target="_blank")
-                | {{donation.creator}}
-            td.text-right
-              | {{donation.usdAmount}} $
-        b-button(variant="outline-secondary", size="sm", to="/activity")
-          | {{"Don't Show all"}}
+      b-card.p-2.bt-0(no-body)
+        donation-history-component(:limit="10")
+        b-button(variant="outline-secondary", size="sm", to="/donations")
+          | {{"Show all"}}
```

Comment:
Is there some clean/easy way to hide this button when there are no items to show? Put it in the component and don't render it if we're on the dedicated page || items === 0 ?